### DISCUSSION TITLE
It is safe to use `getSize`?

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#3909] [Q&A] [answered]
It is safe to use `getSize` function for cases when user have similar cases as you with custom component creation?
Why this function is not documented? I previously (in v5) have used `theme.fn.size`, now I need to huge refactor my codebase with `getSize` function usage, but I need to know how stable it is?
Many thanks!

**Comment by rtivital** [comment, marked as answer]
Everything that is not documented is not safe to use. All parts of Mantine that are not documented may change without any notice. It will be safer to copy and paste such utilities into your project.

**Reply by 7iomka** [reply]
Thank you. I will copy it locally. But it's very strange that you didn't include such a useful feature in the list of documented.
